### PR TITLE
Add new releases page

### PR DESCRIPTION
**Pull request by benoitgrelard**
This includes the last 2 releases portrayed accurately, I will retroactively fill the older releases later.

### CODE REVIEW

**Review by peduarte — APPROVED** [review]

**Comment by peduarte** [comment]
Direct link https://website-git-releases-radix-ui.vercel.app/primitives/docs/overview/releases ./cc @colmtuite

File: pages/primitives/docs/overview/releases.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Releases
+description: Find below all Radix Primitives releases, with their changelogs.
```
Comment (jjenzz) [inline pages/primitives/docs/overview/releases.mdx:3]: 🙏  

```suggestion
description: Radix Primitives releases and their changelogs.
```

Obviously feel free to ignore if you disagree!

File: pages/primitives/docs/overview/releases.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+navRank: 7
```
Comment (jjenzz) [inline pages/primitives/docs/overview/releases.mdx:4]: I can't see `Releases` in the nav for some reason at the preview link. Are there some caching issues perhaps?

**Comment by jjenzz** [comment]
Looks great btw 🎉

File: pages/primitives/docs/overview/releases.mdx (hunk 2 above)
Comment (benoitgrelard) [reply pages/primitives/docs/overview/releases.mdx:4]: Yeah it's the same problem, I'll bust it once we merge.

**Pull request merged**